### PR TITLE
Upgrade dependencies (Storybook v6.2.1)

### PR DESCRIPTION
**Pull request by j3rem1e**
Storybook v6.2.1
Svelte 3.35.0
Svelte-loader 3.0.0

### CODE REVIEW

**Comment by mvolkmann** [comment]
I'm blocked on being able to use Storybook in a Svelte project created with Vite until this is merged.

**Comment by alexgagnon** [comment]
Any reason why this hasn't been merged yet? Can we do something to help?

**Review by shilman — APPROVED** [review]
LGTM!

**Pull request merged**
